zng/fuser/fuser.go

```diff
@@ -0,0 +1,173 @@
+// Packaage fuser transforms records to a unified schema.
+package fuser
+
+import (
+	"errors"
+
+	"github.com/brimsec/zq/proc/spill"
+	"github.com/brimsec/zq/zcode"
+	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+// Fuser buffers records written to it, assembling from them a unified schema of
+// fields and types.  Fuser then transforms those records to the unified schema
+// as they are read back from it.
+type Fuser struct {
+	zctx        *resolver.Context
+	memMaxBytes int
+
+	spiller  *spill.File
```

Comment:
nit: sort this & below field names.

Reply:
Done in 7f31208919e2c0013640c91752be292e36a5502f.